src/pages/studyView/addChartButton/AddChartButton.tsx

```diff
@@ -99,12 +113,34 @@ class AddChartTabs extends React.Component<IAddChartTabsProps, {}> {
         super(props, context);
         makeObservable(this);
         this.tabsDivRef = React.createRef<HTMLDivElement>();
+
+        // Using the first profile to initialze genericAssayProfileOptionsByType
```

Comment:
just want to make sure, is it the first profile or entity?

Reply:
This should be the first profile, we want to generate entity options based on a profile selection, so use the first profile as the default.